test/integration/governance/GovernorDAO.sol

```diff
@@ -29,91 +29,92 @@ contract IntegrationTestGovernorDAO is BaseTest {
         super.setUp();
     }
 
+    /// TODO Testing currently disabled due to a change in the use of HoldingDeposit for tokens
     /// @notice Test and raise a DAO proposal to send funds
-    function testRaiseAndVoteOnDAOProposal() public {
```

Comment:
This does raise a good point - how are we going to manage the community pool spend? Perhaps those would be `ERC20HoldingDeposits` in their own right....

Reply:
I guess once we get to needing that requirement and have a much firmer idea on whats needed we can work it out then.